python/src/main/python/drivers/piglit-worker.py

```diff
@@ -224,21 +232,26 @@ def do_image_job(image_job: tt.ImageJob, work_dir: str) -> tt.ImageJobResult:
     return res
 
 
-def run_image_job(frag_file: str, json_file: str, status_file: str,
+def run_image_job(json_file: str, status_file: str,
                   png_file: str, output_dir: str, skip_render: bool):
     """
     Runs an image job. Converts the shader job to a piglit shader_test file, then delegates to
     run_shader_test to render with shader_runner. Writes the status of the job to file.
-    :param frag_file: The fragment shader to convert and run.
     :param json_file: The JSON uniforms to use with the shader.
     :param status_file: The status file to write to.
     :param png_file: The PNG file to write to.
     :param output_dir: The directory to use for the job.
     :param skip_render: whether to skip rendering or not.
     """
 
+    use_catchsegv = True
+    # noinspection PyBroadException
+    try:
+        gfuzz_common.tool_on_path('catchsegv')
+    except BaseException:
```

Comment:
Please use `Exception` instead of `BaseException` (and you can then remove the above # noinspection).

Reply:
BroadException inspection still happens if I use Exception instead of BaseException, but yeah, good catch.